store/src/java/com/zimbra/cs/mailbox/Mailbox.java

```diff
@@ -5851,6 +5863,48 @@ public AddInviteData addInvite(OperationContext octxt, Invite inv, int folderId,
         }
     }
 
+    public String getMsgHtmlDesc(ParsedMessage pm) throws ServiceException {
```

Comment:
Technically this function can return null but seems like the callers are never checking for null return and going ahead with value settings.
Example:
```
if (firstInvite.getXZimbraDescriptionHtml() == null) {
	firstInvite.setXZimbraDescriptionHtml(mbox.getMsgHtmlDesc(pm));
}
```
There are many ways to better handle it.

Reply:
Good to add some descriptive comment.

Reply:
done